app/screens/sidebar/postalCodeFilter/postalCodeFilterSelector.js

```diff
@@ -0,0 +1,16 @@
+import unique from 'lodash/sortedUniq';
```

Comment:
- [ ] I would name this import `sortedUniq` to avoid confusion with the lodash `uniq` function.